$NetBSD: patch-ao,v 1.1 2001/09/25 13:31:18 tron Exp $

From the GNOME CVS repository:

        * gui/dialogs/meeting-page.c (build_etable): Updated this to match
        the new ETableSimple interface.

--- calendar/gui/dialogs/meeting-page.c.orig	Wed Aug 15 03:13:37 2001
+++ calendar/gui/dialogs/meeting-page.c	Tue Sep 25 13:59:27 2001
@@ -1306,18 +1306,21 @@
 	/* The table itself */
 	priv->model = e_table_simple_new (column_count,
 					  row_count,
+					  append_row,
+
 					  value_at,
 					  set_value_at,
 					  is_cell_editable,
+
+					  NULL,
+					  NULL,
+
 					  duplicate_value,
 					  free_value,
 					  init_value,
 					  value_is_empty,
 					  value_to_string,
 					  mpage);
-	gtk_object_set (GTK_OBJECT (priv->model),
-			"append_row", append_row,
-			NULL);
 	
 	priv->etable = e_table_scrolled_new_from_spec_file (priv->model,
 							    extras, 
